### DISCUSSION TITLE
Cancel cutoff only days

### DISCUSSION THREAD

**Discussion by MichaelWybo** [liedekef/events-made-easy#517] [General]
Hello Franky

In the RSVP, we can choose a cutoff in days

![image](https://github.com/liedekef/events-made-easy/assets/32988994/1e7f8027-b7c7-4658-91b7-bd776c0e5689)

Should it be only an integer or can it be a float number so we can implement a cutoff in hours instead of days ? Like 3h before stard of the event .

**Comment by liedekef** [comment]
It can be a floating point number too (just try it, EME will correct your input if wrong anyway :-) )

**Reply by MichaelWybo** [reply]
Just tried it trunk the float into integer
0,2 --> 0
1.2 --> 1

---
Another question, if there is a waiting_list with auto transfer to the confirmed list in case of cancellation, does it check the 'rsvp_end_number_hours' property of the event before transfering the booking from the watinglist to the confirmed list

**Comment by liedekef** [comment]
indeed, the converrsion to int is a bug, this fixes it: https://github.com/liedekef/events-made-easy/commit/0bf38ffc56eb326efbf36a3e1e3aaea896a97488#diff-30337f1439bbe9482e66c6be5abf33ff6279fcc5ffaef7db9b67967aab883a07
(will be in next version)
The waiting list will automatically move a booking to the approved list if either the price to pay is 0 or no approval is needed. It does not check the rsvp_end* properties since the booking is already made.
